source/languages.ttl

```diff
@@ -2530,8 +2530,7 @@
 
 <sju> a schema:Language ;
     owl:sameAs <9su> ;
```

Comment:
Maybe add a `skos:historyNote` (or just an `rdfs:comment`) to make it clear that once it time, `"9su"` was used as a code for this? (An `owl:sameAs` is "opaque" and won't be accessible in UI views.)

Reply:
Reply